test/sly-package-fu-tests.el

```diff
@@ -0,0 +1,50 @@
+;; -*- lexical-binding: t; -*-
+(require 'sly-package-fu "contrib/sly-package-fu")
+(require 'sly-tests "lib/sly-tests")
+
+(def-sly-test sly-package-fu-sly-import (initial-defpackage final-defpackage symbol)
+  "Check if importing `import` on `initial-defpackage` results in `final-defpackage."
+  '(("(defpackage :foo
+  (:use :cl))
+(in-package :foo)"
+     "(defpackage :foo
+  (:use :cl)
+  (:import-from #:cl
+                #:find))
+(in-package :foo)"
+     "cl:find")
+    ("(defpackage :foo
+  (:use :cl)
+  (:import-from #:cl
+                #:find))
+(in-package :foo)"
+     "(defpackage :foo
+  (:use :cl)
+  (:import-from #:cl
+                #:position
+                #:find))
+(in-package :foo)"
+     "cl:position")
+    ("(defpackage :foo
+  (:use :cl)
+  (:import-from #:bknr.datastore-dummy
+                #:find))
+(in-package :foo)"
+     "(defpackage :foo
+  (:use :cl)
+  (:import-from #:bknr.datastore-dummy
+                #:position
```

Comment:
This is the test that fails before this change. Before this change it will result in two separate `:import-from`s